### PR TITLE
Fix page headers of material, forum and topic

### PR DESCRIPTION
**Pull request by xzhflying**

### CODE REVIEW

File: app/views/course/forum/forums/show.html.slim [hunk 1]
```diff
@@ -1,11 +1,9 @@
 = page_header format_inline_text(@forum.name) do
-
-= render 'controls'
+  = render 'controls'
 
 table.table.forum-list.table-hover
   thead
     tr
-      th
```
Comment (lowjoel) [inline app/views/course/forum/forums/show.html.slim]: why were there 6?

**Comment by xzhflying** [comment]
@lowjoel

File: app/views/course/forum/forums/show.html.slim (hunk 1 above)
Comment (xzhflying) [inline app/views/course/forum/forums/show.html.slim]: hmm... oh wait. It should be 6. The first th should be some icons like unread or something else.

Comment (xzhflying) [inline app/views/course/forum/forums/show.html.slim]: what do you think? @lowjoel

Comment (lowjoel) [inline app/views/course/forum/forums/show.html.slim]: just count correctly and account for every column

Comment (xzhflying) [inline app/views/course/forum/forums/show.html.slim]: OK, I'll check by hand.

**Pull request merged**